components/infobox/wikis/warcraft/infobox_league_custom.lua

```diff
@@ -235,19 +250,28 @@ function CustomLeague._getGameVersion()
 	}
 end
 
+---@param args table
+---@return boolean
 function CustomLeague:shouldStore(args)
 	return Namespace.isMain() and
 		not Logic.readBool(Variables.varDefault('disable_LPDB_storage', 'false'))
 end
 
+---@return string?
+---@return string?
 function CustomLeague:_getServer()
 	if String.isEmpty(_args.server) then
 		return nil
 	end
 
-	return '[[Server|' .. _args.server .. ']]'
+	if String.isEmpty(_args.server2) then
+		return '[[Server|' .. _args.server .. ']]'
+	end
+
+	return '[[Server|' .. _args.server .. ']]', '[[Server|' .. _args.server2 .. ']]'
```

Comment:
Could do

```return table.concat(Array.map(self:getAllArgsForBase(args, 'server'), function(server) return '[[Server|'.. server ..']' end), ', ')```

Reply:
would return a string instead of 2 but will use it without the concat
(also self is not available here, so need to use `_league`)